pkg/cli/admin/catalog/mirror.go

```diff
@@ -386,6 +389,15 @@ func (o *MirrorCatalogOptions) Run() error {
 	return WriteManifests(o.IOStreams.Out, o.SourceRef, o.DestRef, o.ManifestDir, o.IcspScope, mapping)
 }
 
+func (o *MirrorCatalogOptions) PostRun() error {
+	dir := strings.Split(o.DatabasePath, ":")
+	if err := os.RemoveAll(dir[1]); err != nil {
```

Comment:
I think we only want to do this if `--path` is not set

Reply:
I added new o.TempDir, which I set to true if o.DatabasePath is "" meaning --path unset. If o.DatabasePath is set using --path, then o.TempDir remains false.
At the end based if the o.TempDir is set or not we decide to delete the directory.